test/integration/test_server.js

```diff
@@ -0,0 +1,40 @@
+const path = require('path');
+const request = require('request');
+const test = require('tape');
+
+const lib = require('../../lib');
+
+const HOST = process.env.TARGET_HOST || 'http://localhost:8080';
```

Comment:
It seems like this `TARGET_HOST` env var would also be useful for folks running `dinghy` or docker-machine or other uncommon docker setups; maybe it's worth documenting in the readme?